modules/instance/versions.tf

```diff
@@ -20,15 +20,15 @@ terraform {
 
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.53, < 5.0"
+      version = ">= 3.53, < 6"
     }
     null = {
       source  = "hashicorp/null"
       version = ">= 2.0"
     }
     template = {
-      source  = "hashicorp/template"
-      version = "~> 2.0"
+      source  = "hashicorp/template" # TODO: this provider is deprecated, replace with terraform 0.13+ templatefile function
+      version = "2.2.0"
```

Comment:
While `2.2.0` is pretty old, increasing the minimum provider version could be construed as a breaking change?

Reply:
It was already downloading 2.2 since it's been out for 3 years. I don't think it'll break anyone.